File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        for m,i in enumerate(nums):
            for n,j in enumerate(nums):
                if i+j==target and n!=m:
                    return [m,n]
                    

       
            

           

            
        
            
        
            

        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target)
    {
        int l;
        l=nums.size();
        for (int i=0 ; i<l; i++)
        {
            for (int j=i+1 ; j<l; j++)
            {
                int s = nums[i]+nums[j];
                if (s==target)
                {
                

                  return {i,j};

                }
            
            }
        }
        return {};
        
    }
};